client/landing/stepper/declarative-flow/internals/components/boot/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useLocale } from '@automattic/i18n-utils';
+import {
+	ComponentProps,
```

Comment:
Should this be imported as a type?
```suggestion
	type ComponentProps,
```